src/helpers.py

```diff
@@ -84,6 +85,22 @@ def download_manifest(deck: dict, access_token: str, environment_index: int = 0)
                 f"and save a valid values path."
             )
             exit(1)
+        elif e.response.status_code == 403:
+            console.warning("Refreshing access token")
+            environment_id = deck["environment"][environment_index]["id"]
+            response = authentication.refresh()
+            if not response["success"]:
+                console.exit_login_required()
```

Comment:
In case the refresh is not working (and there no other HTTP error) it means the user really has to log in again, please add the corresponding error message.